pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -2895,8 +2895,8 @@ public bool ImportTool (string title, string command, string arguments, string i
 //                        _out.WriteLine("         tool {0} was not modified.", tool.Title);
                         return true;
                     }
-                    // Ovewrite conflicts
-                    if (resolveToolConflictsBySkipping == false)
+                    // Overwrite conflicts

```

Comment:
The ifs above are a bit strange as well. I would use:

if (r.IsNull)
else if (r.Value)
else

Though, you could also drop the elses, since the blocks all have a return. But it should be consistent.